assets/components/contributionSelection/helpers.js

```diff
@@ -47,6 +33,12 @@ const amounts = {
       { value: '250', spoken: 'two hundred and fifty' },
       { value: '500', spoken: 'five hundred' },
     ],
+    EUR: [
+      { value: '25', spoken: 'twenty five' },
+      { value: '50', spoken: 'fifty' },
+      { value: '100', spoken: 'one hundred' },
+      { value: '250', spoken: 'two hundred and fifty' },
```

Comment:
Why are these numbers -> spoken indexed by by currency when they do not vary by currency?

Reply:
There is huge duplication around this logic, I deleted a [little of it](https://github.com/guardian/support-frontend/pull/492/files/f344d1d5b9d977a5c888a00460a0468586491846#diff-6c1e5e462dbe45539d3684a4258bcb50L15) but there is a lot of work to be done. I prefer to keep that work separated of this PR.

Reply:
They don't vary by currency just now, but there's a good chance they will when the contributions team start running amounts tests on them.